packages/react-code-editor/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -363,7 +363,7 @@ export class CodeEditor extends React.Component<CodeEditorProps, CodeEditorState
 
   componentDidMount() {
     document.addEventListener('keydown', this.handleGlobalKeys);
-    this.observer = getResizeObserver(this.ref.current, this.handleResize);
+    this.observer = getResizeObserver(this.ref.current, this.handleResize, true);
```

Comment:
Since useRequestAnimationFrame is opt-in, thinking we should leave this off the examples. (Don't feel strongly about it, but suspect we probably don't need it?) That is, unless there is an issue with the resize observer not working here?